src/hotspot/share/utilities/vmError.hpp

```diff
@@ -251,4 +251,41 @@ class VMErrorCallbackMark : public StackObj {
   ~VMErrorCallbackMark();
 };
 
+// Ergonomic construction for creating ad-hoc VMErrorCallback which automatically
```

Comment:
Does `Ergonomic construction` have specific meanings in cpp/this context? If not, I wonder if `lightweight` is clearer, as it is less technical.